local-volume/provisioner/pkg/deleter/deleter.go

```diff
@@ -26,6 +26,12 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
+const (
+	// kubelet event reason list
+	FailedCleaningLocalPV = "FailedCleaningLocalPV"
```

Comment:
@wongma7 @jsafrane what do external provisioners do when the deletion fails?  I'm thinking we may need to add a new event "FailedDeleteVolume"